libcxx/test/std/numerics/simd/simd.class/subscript.assert.pass.cpp

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Not attached to this file: let's add a test for `operator[]` even if it looks silly/redundant, just for completeness. And in that test we can validate details like the fact that the method is const-qualified, etc.